CHANGELOG.md

```diff
@@ -7,6 +7,9 @@
 - Technical - Adapt release manager to conventional commits.
 - Documentation - Update links in readme.
 
+### Fixed
+- Command Generate - Fix DATABASE_URL badly encoded in .env generation.
```

Comment:
The whole content is not encoded anymore by Handlebars, not just `DATABASE_URL`. I don't have a suggestion though (I don't like the one I suggested).

```suggestion
- Command Generate - Fix bad encoding of content in generated files (remove HTML escaping in template file).
```